server/services/gameLogic.js

```diff
@@ -38,7 +38,8 @@ let cards = [];
 let votes = [];
 
 /* Return the list of players, without their assigned cards */
-exports.getPlayers = () => players.map(player => ({ username: player.username, score: player.score }));
+const getPlayers = () => players.map(player => ({ username: player.username, score: player.score }));
+exports.getPlayers = getPlayers;
```

Comment:
This can be one line `const getPlayers = exports.getPlayers = () => players.map(player => ({ username: player.username, score: player.score }));`

Reply:
Ironically, it was my comment that switched it from a one liner. I think this way looks a little cleaner but that's a personal preference.

Reply:
Oh haha fair enough, I just saw someone do it online as a one-liner but I'm happy to do two if you think it's nicer